### ISSUE TITLE
Add black code formatting precommit

### ISSUE THREAD

**Issue by MaxGhenis** [UBICenter/openfisca-us#37]

**Comment by martinholmer** [comment]
@MaxGhenis,  over a year ago your raised issue #37 that has this title:
"Add black code formatting precommit"

I notice that the current Makefile target `format` runs `black` on the code.
Is this what you meant by "precommit"?  Or do you have something else in mind?

**Comment by MaxGhenis** [comment]
No, precommit hooks differ from Makefiles: https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/

**Comment by DTrim99** [comment]
@MaxGhenis Closing as obsolete — the repo moved from black to ruff for formatting (make format runs ruff format; CI enforces ruff format --check), so a black precommit is no longer applicable. Reopen if a precommit hook is still wanted.
